examples/BUILD

```diff
@@ -53,7 +53,7 @@ android_library(
     srcs = glob(["handwritten-src/java/com/dropbox/textsort/*.java"]),
     custom_package = "com.dropbox.textsort",
     manifest = "android/app/src/main/AndroidManifest.xml",
-    resource_files = glob(["android/app/src/main/res/**"]),
+    resource_files = glob(["android/app/src/main/res/**"], exclude=["android/app/src/main/res/.DS_Store"]),
```

Comment:
Does this cause builds to fail?

Reply:
yeah, basically if Mac Os adds this .DS_Store files, then the glob is picking them up and failing the build.

Reply:
This is strange.  Do you mean Macos will put files in a source tree that cause bazel based projects to fail?

Reply:
Exactly